src/esmockLoader.mjs

```diff
@@ -15,11 +15,12 @@ const urlDummy = 'file:///' + path
   .join(path.dirname(url.fileURLToPath(import.meta.url)), 'esmock.js')
   .replace(/^\//, '');
 
+const esmockGlobalsAndAfterRe = /\?esmockGlobals=.*/;
 const esMockGlobalsAndBeforeRe = /.*\?esmockGlobals=/;
 const esmockModuleKeysRe = /#esmockModuleKeys/;
 const exportNamesRe = /.*exportNames=(.*)/;
+const whitespaceRe = /%20/g;
```

Comment:
not very familiar with the loader, but does it make sense to use `decodeURI("file://xxx")`, instead of hard-code `%20`?

Reply:
great idea! thank you